plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlJdbcConnectionCreation.java

```diff
@@ -62,7 +62,9 @@ protected QueryRunner createQueryRunner()
                     runner.createCatalog("counting_postgresql", "counting_postgresql", ImmutableMap.of(
                             "connection-url", postgreSqlServer.getJdbcUrl(),
                             "connection-user", postgreSqlServer.getUser(),
-                            "connection-password", postgreSqlServer.getPassword()));
+                            "connection-password", postgreSqlServer.getPassword(),
+                            // connection reuse hides the actual number of I/O operations done
```

Comment:
When this test was written, it was the case to see that reuse reuses the connection. Even test name stresses that "ConnectionCreation", here what you do is metadata call counting. Which is a bit different, maybe you need a different test?

Reply:
Good point. I did check and there was `TestReusableConnectionFactory` but that's only a unit test.